content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -72,213 +114,233 @@ Once the :guilabel:`Lot/Serial Number` is generated, click the blank field next
-:guilabel:`Date`, and the :guilabel:`Lot/Serial #` assigned.
-
-The :guilabel:`Traceability Report` can also include a :guilabel:`Reference` receipt from the
-previous purchase order (PO), if any of the product quantities shared a serial number assigned
-during receipt of that specific :abbr:`PO (purchase order)`.
-
-Manage serial numbers for different operations types
-====================================================
-
-By default in Odoo, the creation of new serial numbers is only allowed upon **receiving** products
-from a purchase order. **Existing** serial numbers cannot be used. For sales orders, the opposite is
-true: new serial numbers cannot be created on the delivery order, only existing serial numbers can
-be used.
-
-To change the ability to use new (or existing) serial numbers on any operation type, go to
-:menuselection:`Inventory app --> Configuration --> Operations Types`, and select the desired
-:guilabel:`Operation Type`.
-
-For the :guilabel:`Receipts` operation type, found on the :guilabel:`Operations Types` page, the
-:guilabel:`Use Existing Lots/Serial Numbers` option can be enabled, by selecting
-:guilabel:`Receipts` from the :guilabel:`Operations Types` page, clicking :guilabel:`Edit`, and then
-clicking the checkbox beside the :guilabel:`Use Existing Lots/Serial Numbers` option (in the
-:guilabel:`Traceability` section). Lastly, click the :guilabel:`Save` button to save the changes.
-
-For the :guilabel:`Delivery Orders` operation type, located on the :guilabel:`Operations Types`
-page, the :guilabel:`Create New Lots/Serial Numbers` option can be enabled, by selecting
-:guilabel:`Delivery Orders` from the :guilabel:`Operations Types` page, clicking :guilabel:`Edit`,
-and clicking the checkbox beside the :guilabel:`Create New Lots/Serial Numbers` option (in the
-:guilabel:`Traceability` section). Be sure to click :guilabel:`Save` to save changes.
-
-.. image:: serial_numbers/serial-numbers-operations-types.png
+.. example::
+   For a receipt with a :guilabel:`Demand` of `3.00` products, one product has already been assigned
+   a serial number in the :guilabel:`Open: Stock move` pop-up window.
+
+   So, in the :guilabel:`Import Lots` pop-up window, two serial numbers, `124` and `125` are
+   assigned to the remaining products by entering the following in the :guilabel:`Lots/Serial
+   numbers` input field:
+
+   .. code-block::
+
+      124
+      125
+
+   The :guilabel:`Keep current lines` option is selected to add these two serial numbers **in
+   addition** to the serial number, `123`, that has already been assigned.
+
+   .. image:: serial_numbers/import-serial.png
+      :align: center
+      :alt: Show example of correctly inputting serial numbers in the text field.
+
+.. _inventory/product_management/moves-button:
+
+Moves smart button
+~~~~~~~~~~~~~~~~~~
+
+Accessible from both receipt and delivery order forms, the :guilabel:`Moves` page shows a detailed
```

Comment:
**Moves** is not the name of the page, it's the name of the smart button— the page is called **Detailed Operations**